lib/ast-converter.js

```diff
@@ -69,8 +70,8 @@ module.exports = (ast, extra) => {
     /**
      * Add the comment nodes to the AST (that were parsed separately in parser.js)
      */
-    if (extra.comment || extra.attachComment) {
-        estree.comments = extra.comments || [];
+    if (extra.comment) {
```

Comment:
I know this was redundant until now, but this is breaking change - prettier currently uses the `attachComment` version and will need to be updated.

I think we should still make the change, but we will need to update it to `Breaking`